Storage/SQL/BrowserTable.swift

```diff
@@ -906,25 +881,25 @@ open class BrowserTable: Table {
         if from < 22 && to >= 22 {
             if !self.run(db, queries: [
                 "DROP TABLE IF EXISTS \(TablePageMetadata)",
-                attachedPageMetadataCreate,
-                attachedIndexPageMetadataCacheKeyCreate,
-                attachedIndexPageMetadataSiteURLCreate]) {
+                pageMetadataCreate,
```

Comment:
Drop these lines altogether. Someone upgrading from 21 will run `pageMetadataCreate` here, then run `highlightsCreate`, then drop and recreate highlights, then in the 27 upgrade will drop page metadata and create it again.